### ISSUE TITLE
Add safety:mask:covid19 field.

### ISSUE THREAD

**Issue by TheAdventurer64** [openstreetmap/iD#7997]
This field would allow users to put down if a place requires you to wear a mask, or not. More info [here](https://wiki.openstreetmap.org/wiki/Key:safety:mask:covid19).

**Comment by matkoniecz** [comment]
It seems weird one, as it

a) changes in entire region quite frequently (hello 2020!)
b) applies to all objects of a given type in a given area (at least in Europe)

So it seems weird to me to check it on the ground. "this shop requires mask, like this one and this one and all other inside this country/state".

**Comment by TheAdventurer64** [comment]
In the US, the country doesn't have a national mask mandate.

**Comment by matkoniecz** [comment]
I may be mistaken but there are mask mandates at lower levels (state, maybe city/county)? Or is it differing from shop to shop?

**Comment by quincylvania** [comment]
> there are mask mandates at lower levels (state, maybe city/county)? Or is it differing from shop to shop?

Yes, some administrative areas in the US may have their own regulations. They do change quite often. Regardless, many businesses have their own rules so this could be a useful tag if data consumers supported it.
